### DISCUSSION TITLE
Status of the 'trace_replayTransaction' RPC method

### DISCUSSION THREAD

**Discussion by itgav** [erigontech/erigon#7807] [Q&A]
What's the status of the `trace_replayTransaction` and `trace_replayBlockTransactions` RPC methods? On the RPC Daemon `README` it says it's implemented but has the notes _"stateDiff only"_. Does that mean the `stateDiff` flag works perfectly but the `trace` and `vmTrace` flags have issues? Are there known issues or limitations with either of the trace methods or the specific flag results for them?

**Comment by AskAlexSharov** [comment]
I don’t know any limitations
